### PR TITLE
Feature/28/user can see which users have liked

### PR DESCRIPTION
**Pull request by jeff1108**

### CODE REVIEW

**Comment by jeff1108** [comment]
Fix the conflict between the development branch and like branch
Add View: User can see which user have like that post

File: app/views/likes/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+All Like User:
```
Comment (elishkaflint) [inline app/views/likes/index.html.erb]: can we changed this to

Post Liked By:

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -28,7 +28,7 @@
       <%= link_to 'Delete', user_post_path(@user.id, post.id), method: "delete" %>
       <%= link_to 'Update', user_post_path(@user.id, post.id), method: "get" %>
     <% end %>
-    <p>Likes: <%= post.likes.length %></p>
+    <p><%= link_to 'Likes:', user_post_likes_path(@user.id, post.id) %> <%= post.likes.length %></p>
```
Comment (elishkaflint) [inline app/views/users/index.html.erb]: Can we please change Likes: to Likes
ie. remove semi-colon

Comment (elishkaflint) [reply app/views/users/index.html.erb]: As discussed in the meeting earlier, is it difficult to add likes to the user profile page as well? If it's more than a 15 min job don't worry - but would be cool to have it there as well.

File: spec/features/user_can_see_which_users_have_liked_post_spec.rb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  before(:each) do
+    @user1 = User.create(first_name: "test_first_name", last_name: "test_last_name", email: "test_email@keepingitrails.com", password: "pa55w0rd")
+    @user2 = User.create(first_name: "test_second_name", last_name: "test_last_name", email: "test_email2@keepingitrails.com", password: "pa55w0rd")
+    @post = Post.create(user_id: @user1.id, message: "Test message")
+  end
+
+  scenario "A user can see his/her like on own post" do
+    successful_sign_in
+    click_button "Like"
+    click_link "Likes:"
```
Comment (elishkaflint) [inline spec/features/user_can_see_which_users_have_liked_post_spec.rb]: Note - if you take the semi-colon off likes as suggested in my comment above, take it off here as well (and below) to make sure tests pass.

**Review by elishkaflint — COMMENTED** [review]
Really good work guys, just a couple of things 👍

**Review by elishkaflint — APPROVED** [review]
🐤

**Pull request merged**